test/parallel/test-os-process-priority.js

```diff
@@ -109,6 +109,16 @@ for (let i = PRIORITY_HIGHEST; i <= PRIORITY_LOW; i++) {
   checkPriority(process.pid, i);
 }
 
+{
+  // From libuv docs
+  // "The priority value range is between -20 (high priority) and 19 (low priority)"
```

Comment:
This comment isn't really relevant here (or there is a bug in the code change). The range mentioned here is the priority that is passed to `setPriority()`, but you're calling `getPriority()`.

The code change here is trying to get the priority of process id = -21. That explains why the error is `ESRCH` - although not on macOS apparently.

Reply:
That makes sense. I will remove the comment.

Replacing -21 w/ -1 (negative numbers in general) should make the same effect; for that case, macOS will always return `EINVAL` as described in the [getpriority() macOS ref](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man2/getpriority.2.html). I will change the expected error to do a "match" instead.